library/linear.lib

```diff
@@ -1564,59 +1699,16 @@ $FPLIST
  Texas*DSBGA*Pitch0.5mm*
 $ENDFPLIST
 DRAW
-X GND A3 0 -400 100 U 50 50 0 0 W
-X VCC A2 0 400 100 D 50 50 0 0 W
 S -350 -300 350 300 0 1 10 f
 S -350 -300 350 300 0 1 10 f
+X VCC A2 0 400 100 D 50 50 0 0 W
+X GND A3 0 -400 100 U 50 50 0 0 W
+X DIS A1 500 0 150 L 50 50 1 1 I
+X THR B1 500 -200 150 L 50 50 1 1 I
+X CV C1 -500 0 150 R 50 50 1 1 I
+X R C2 -500 -200 150 R 50 50 1 1 I I
 X TR B3 -500 200 150 R 50 50 1 1 I
 X Q C3 500 200 150 L 50 50 1 1 O
-X R C2 -500 -200 150 R 50 50 1 1 I I
-X CV C1 -500 0 150 R 50 50 1 1 I
-X THR B1 500 -200 150 L 50 50 1 1 I
-X DIS A1 500 0 150 L 50 50 1 1 I
-ENDDRAW
-ENDDEF
-#
-# LMV321
```

Comment:
LMV321 deleted.
LMV321M7 is now an alias of Amp.S.SC-70-5.Ni-Pn-Ii-O-Pp
LMV321M5 is now an alias of Amp.S.SOT-23-5.Ni-Pn-Ii-O-Pp